packages/core/src/types/ruleGroups.ts

```diff
@@ -7,6 +7,12 @@ export interface CommonRuleAndGroupProperties {
   path?: Path;
   id?: string;
   disabled?: boolean;
+  /**
+   * Whether this rule or group is muted. When muted, the rule or group
+   * is excluded from query export formats (SQL, JSON, MongoDB, etc.).
+   * For groups, muting recursively mutes all children.
```

Comment:
I'm not sure how you implemented this in your org, but my inclination would be to have nested rules/groups inherit the `muted` property of the closest ancestor group where the property was explicitly defined, rather than recursively setting the property itself on all nested objects. The inheritance method is how the existing "lock" functionality works.

Reply:
That's fair. I'm more verbose generally speaking, and an explicit approach felt like a better way to come crashing into the repository 😅 

Will have a change set up shortly.